### ISSUE TITLE
PersonaCoin: Div elements make it unusable inside buttons

### ISSUE THREAD

**Issue by micahgodbolt** [OfficeDev/office-ui-fabric-react#11461]
Breakout of #11431

Personas are often used inside of buttons, but due to persona coins being built with divs they violate the role of buttons only containing phrasing content.
